packages/nameguard-sdk/src/index.ts

```diff
@@ -698,7 +778,9 @@ export class NameGuard {
     options?: SecurePrimaryNameOptions,
   ): Promise<SecurePrimaryNameResult> {
```

Comment:
Please also check the `nameguard-js` package and see if there's anything there that needs to be updated. NameGuard JS implements a subclass of the `NameGuard` class.